packages/react-components/react-message-bar-preview/stories/MessageBar/Animation.stories.tsx

```diff
@@ -91,7 +91,12 @@ export const Animation = () => {
             </MessageBarBody>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29432&commitId=6634e9cbbe3bd8b3999fe70096ddd51b492cfa1a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Layout Fit.normal.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>